README.md

```diff
@@ -29,53 +29,59 @@ Learn how to automate your workflow using Twilio's REST API and Twilio SMS. This
 
 1. Clone this repository and `cd` into it.
 
- ```
- git clone git@github.com:TwilioDevEd/airntg-servlets.git
- ```
+    ```bash
+    $ git clone git@github.com:TwilioDevEd/airntg-servlets.git
```

Comment:
@hortega I think it's of good practice to avoid writing the `$` symbol, because it's easy to copy/paste the code.